src/components/Player/Player.jsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
-import { connect } from 'react-redux';
-import { withRouter } from 'react-router';
+import {
+  connect,
+} from 'react-redux';
```

Comment:
My autoformatter automatically does this.  I'm personally okay with it because it makes it easier to add to if we ever add a second import.  It still passes eslint